inv_kin_node:
  ros__parameters:
    lx: 0.3  # Override node default
    ly: 0.3  # Override node default
    wheel_radius: 0.04  # Override node default
    cmd_vel_topic: '/custom_cmd_vel'
    motor_vel_topic: '/custom_motor_vel'
    frame_id: 'custom_base_link'

